server/protocols/webadmin/webadmin-mailrepository/src/test/java/org/apache/james/webadmin/routes/MailRepositoriesRoutesTest.java

```diff
@@ -88,7 +88,6 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Sets;
-
```

Comment:
I see so many changes like that. Are they really needed? Are there in a specific commit?

Reply:
they are in the commit which move the class Mailbox from store to api. 
I re-organized the imports on all project. It was a bit overkill maybe...